File: thitracnghiemapp/lib/models/chu_de.dart
class ChuDe {
  final int id;
  final String tenChuDe;
  final String? moTa;

  const ChuDe({required this.id, required this.tenChuDe, this.moTa});

  factory ChuDe.fromJson(Map<String, dynamic> json) {
    return ChuDe(
      id: json['id'] as int,
      tenChuDe: json['tenChuDe'] as String? ?? '',
      moTa: json['moTa'] as String?,
    );
  }

  Map<String, dynamic> toJson() {
    return {'id': id, 'tenChuDe': tenChuDe, 'moTa': moTa};
  }
}

File: ThiTracNghiemApi/Models/ChuDe.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace ThiTracNghiemApi
{
    public class ChuDe
    {
        [Key]
        public int Id { get; set; }
        [Required]
        public string TenChuDe { get; set; } = string.Empty;
        public string MoTa { get; set; } = string.Empty;
        public ICollection<CauHoi> CauHois { get; set; } = new List<CauHoi>();
        public ICollection<DeThi> DeThis { get; set; } = new List<DeThi>();
    }
}